### PR TITLE
fixes some null refs

### PR DESCRIPTION
**Pull request by aersamkull**
We could not compile because of these Null-Refs. The fixes are trivial, as seen below

### CODE REVIEW

**Comment by aersamkull** [comment]
The error occurs when using: `[DefaultValue(null)]`

File: src/Swashbuckle.AspNetCore.SwaggerGen/Generator/OpenApiAnyFactory.cs [hunk 1]
```diff
@@ -25,14 +25,14 @@ public static bool TryCreateFor(OpenApiSchema schema, object value, out IOpenApi
             else if (schema.Type == "number" && schema.Format == "double" && TryCast(value, out double doubleValue))
                 openApiAny = new OpenApiDouble(doubleValue);
 
-            else if (schema.Type == "string" && value.GetType().IsEnum)
+            else if (schema.Type == "string" && value !=null && value.GetType().IsEnum)
```
Comment (EvgeniyZ) [inline src/Swashbuckle.AspNetCore.SwaggerGen/Generator/OpenApiAnyFactory.cs]: I think it's better to format it, add whitespace before null

Comment (aersamkull) [reply src/Swashbuckle.AspNetCore.SwaggerGen/Generator/OpenApiAnyFactory.cs]: Done :)

**Comment by domaindrivendev** [comment]
No longer neccessary

**Pull request closed without merging**